liquibase-cdi-jakarta/src/main/java/liquibase/integration/jakarta/cdi/CDIBootstrap.java

```diff
@@ -85,11 +78,6 @@ public boolean isAlternative() {
                 return false;
             }
 
-            @Override
```

Comment:
May I inquire as to why this method has been removed, @filipelautert ?

Reply:
It was removed because it was removed upstream - so it's "overridable" anymore.

Reply:
Thanks for the explanation, @filipelautert.